src/ir/function.rs

```diff
@@ -597,6 +597,10 @@ impl ClangSubItemParser for Function {
             return Err(ParseError::Continue);
         }
 
+        if cursor.is_deleted_function() {
+            return Err(ParseError::Continue);
+        }
```

Comment:
It might be worth to shuffle the logic a bit to do less work in the common case, something like:

```rust
if cursor.is_inlined_function() {
    if !context.options().generate_inline_functions {
        return Err(..);
    }
    if cursor.is_deleted_function() {
        return Err(..);
    }
}
```